nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -1176,6 +1176,148 @@ Secure Prompt
 
 If you prefer not to provide the password or raw key in the command-line invocation of the tool, leaving these arguments absent will prompt a secure console read of the password (by default) or raw key (if the `-r` flag is provided at invocation).
 
+[[admin-toolkit]]
+Administrative Tools
+--------------------
+The admin toolkit contains command line utilities for administrators to support NiFi maintenance in standalone
+and clustered environments. These utilities include:
+
+* Notify -- The notification tool allows administrators to send bulletins to the NiFi UI using the command line.
+* Node Manager -- The node manager tool allows administrators to safely disconnect or remove nodes from a NiFi cluster.
+
+The admin toolkit is bundled with the nifi-toolkit and can be executed with scripts found in the _bin_ folder.
+
+=== Prerequisites for Running Admin Toolkit in a Secure Environment
+For secured nodes and clusters, two policies should be configured in advance:
+
+* Access the controller – A user that will have access to these utilities should be authorized in NiFi by creating an “access the controller” policy (/controller) with both view and modify rights.
+* Proxy user request – If not previously set node’s identity ( the DN value of the node’s certificate) should be authorized to proxy requests on behalf of a user
+
+When executing either the notify or node manager tools in a secured environment the proxyDN flag option should be used in order to properly identify the user that was authorized to execute these commands. In non-secure environments the flag will be ignored.
+
+=== Notify
+Notify allows administrators to send messages as bulletins to NiFi.  Notify is supported on NiFi version 1.2.0 and higher.
+The notification tool is also available in a notify.bat file for use on Windows machines.
+
+
+
+To send notifications:
+
+ notify.sh -d {nifi root folder} –b {nifi bootstrap file path} -m {message} [-l {level}][-v]
+
+
+To show help:
+
+ notify.sh -h
+
+The following are available options:
+
+* -b  | --bootstrapConf - Existing Bootstrap Configuration file (required)
```

Comment:
Annoying, but to be consistent, the flags listings above have each flag denoted as code:

* `-b`  | `--bootstrapConf` - Existing...

Actually, it's a slightly different format. If going for consistency:

* `-b`,`--bootstrapConf <arg>`                  The bootstrap.conf file to persist master key